src/naive/internals.rs

```diff
@@ -371,7 +371,7 @@ impl fmt::Debug for Of {
 /// (month, day of month and leap flag),
 /// which is an index to the `MDL_TO_OL` lookup table.
 #[derive(PartialEq, PartialOrd, Copy, Clone)]
-pub(super) struct Mdf(pub(super) u32);
+pub(crate) struct Mdf(pub(super) u32);
```

Comment:
The thing I see is that `Mdf` needs to be exposed, so I am not sure if it is a good tradeoff is the internals are being exposed.